statestorage/statestorage_mgr.go

```diff
@@ -55,25 +55,28 @@ func fileExists(filename string) bool {
 	return !info.IsDir()
 }
 
-func InitDb(dbFile string, isNewDB bool) {
-	if (isNewDB == true && fileExists(dbFile)) {
+func InitNewDb(dbFile string) {
+	if (fileExists(dbFile)) {
 		fmt.Printf("\ndataStorageMgr: DB %s already exist, cannot create a new with same name.\n", dbFile)
 		os.Exit(1)
-	} else if (fileExists(dbFile)) {
+	}
+	db, dbErr = sql.Open("sqlite3", dbFile) // Creates a new dbFile(?)
+	checkErr(dbErr)
+	err := createStaticTables()
+	if err != 0 {
+		fmt.Printf("\novdsServer: Unable to make static tables : %s\n", err)
+		os.Exit(1)
+	}
+}
+
+func InitExistingDb(dbFile string) {
+	if (fileExists(dbFile)) {
 		db, dbErr = sql.Open("sqlite3", dbFile)
 		checkErr(dbErr)
-		if (isNewDB == true) {
- 		    err := createStaticTables()
-		    if err != 0 {
-			    fmt.Printf("\novdsServer: Unable to make static tables : %s\n", err)
-			    os.Exit(1)
-		    }
-		}
 	} else {
```

Comment:
(old code)
The only remaining branch is here, so this is where we end up for NewDB = true and file does not exist.